kmip/core/factories/secrets.py

```diff
@@ -68,7 +69,7 @@ def create(self, secret_type, value=None):
             SymmetricKey(...)
         """
         if secret_type is ObjectType.CERTIFICATE:
-            return self._create_certificate(value)
+            return self._create_certificate()
```

Comment:
Oh, but the `store_secret` method in the `kmip_secret_store` in Barbican currently only goes through the `SecretFactory` to create secrets. Are you going to branch out to handle certificates differently? If you're going to use `store_secret` for certificates, you'll need to change things around. Certificates don't have algorithms or cryptographic lengths.